src/fibsem_tools/io/h5.py

```diff
@@ -7,78 +7,116 @@
 
 H5_ACCESS_MODES = ("r", "r+", "w", "w-", "x", "a")
 
-H5_DATASET_KWDS = ("name",
-                     "shape",
-                     "dtype",
-                     "data",
-                     "chunks",
-                     "compression",
-                     "compression_opts",
-                     "scaleoffset",
-                     "shuffle",
-                     "fletcher32",
-                     "maxshape",
-                     "fillvalue",
-                     "track_times",
-                     "track_order",
-                     "external",
-                     "allow_unknown_filter")
-
-H5_GROUP_KWDS = ("name",
-                "track_order")
-
-H5_FILE_KWDS = ("name",
-                  "mode",
-                  "driver",
-                  "libver",
-                  "userblock_size",
-                  "swmr",
-                  "rdcc_nslots",
-                  "rdcc_nbytes",
-                  "rdcc_w0",
-                  "track_order",
-                  "fs_strategy",
-                  "fs_persist",
-                  "fs_threshold")
+H5_DATASET_KWDS = (
+    "name",
+    "shape",
+    "dtype",
+    "data",
+    "chunks",
+    "compression",
+    "compression_opts",
+    "scaleoffset",
+    "shuffle",
+    "fletcher32",
+    "maxshape",
+    "fillvalue",
+    "track_times",
+    "track_order",
+    "dcpl",
+    "external",
+    "allow_unknown_filter",
+)
+
+H5_GROUP_KWDS = ("name", "track_order")
```

Comment:
Since `track_order` is a common keyword to Datasets, Group's, and Files perhaps this parameter should be passed to all objects when they are created.